File: Customer Landing Page/patch_navbar.js
const fs = require('fs');

const path = 'src/pages/HelpCenter/components/Navbar.jsx';
let content = fs.readFileSync(path, 'utf8');

// replace navItems
content = content.replace(/const navItems = \[[\s\S]*?\];/, `const navItems = [
  { label: "Home", to: "/" },
  { label: "Help Center", to: "/help-support" },
  { label: "FAQs", to: "/faq" },
  { label: "Contact", to: "/customer-support" },
];`);

// replace <a ... > with <Link to={item.to} ...>
content = content.replace(/<a\s+key=\{item\.label\}\s+href=\{item\.href\}\s+onClick=\{\(e\) => scrollToSection\(e, item\.href\)\}/g, '<Link\n              key={item.label}\n              to={item.to}\n              onClick={() => setMobileMenuOpen(false)}');
content = content.replace(/<\/a>/g, '</Link>');

// Wait, the "Support" button has href="#contact"
content = content.replace(/<a\n\s+href="#contact"[\s\S]*?onClick=\{\(e\) => scrollToSection\(e, "#contact"\)\}/g, '<Link\n            to="/customer-support"');
content = content.replace(/<a\n\s+href="#contact"\n\s+onClick=\{\(e\) => scrollToSection\(e, "#contact"\)\}/g, '<Link\n                to="/customer-support"');

fs.writeFileSync(path, content);

File: Customer Landing Page/patch_navbar.py
import sys, re

path = 'src/pages/HelpCenter/components/Navbar.jsx'
with open(path, 'r') as f:
    content = f.read()

new_nav_items = """const navItems = [
  { label: "Home", to: "/" },
  { label: "Help Center", to: "/help-support" },
  { label: "FAQs", to: "/faq" },
  { label: "Contact", to: "/customer-support" },
];"""

content = re.sub(r'const navItems = \[.*?\];', new_nav_items, content, flags=re.DOTALL)

# replace <a> in loop with <Link>
content = re.sub(r'<a\s+key=\{item\.label\}\s+href=\{item\.href\}\s+onClick=\{\(e\) => scrollToSection\(e, item\.href\)\}',
    r'<Link\n              key={item.label}\n              to={item.to}\n              onClick={() => setMobileMenuOpen(false)}', content)

# replace </a> with </Link> (since we changed the <a key={item.label} ... to <Link)
# Wait, some <a> are not in the loop. The "Support" buttons:
# 1. Desktop Support Button
content = re.sub(r'<a\s+href="#contact"\s+onClick=\{\(e\) => scrollToSection\(e, "#contact"\)\}', r'<Link\n            to="/customer-support"', content)

# 2. Mobile Support Button
content = re.sub(r'<a\s+href="#contact"\s+onClick=\{\(e\) => scrollToSection\(e, "#contact"\)\}', r'<Link\n                to="/customer-support"', content)

content = content.replace('</a>', '</Link>')

with open(path, 'w') as f:
    f.write(content)
